pandas/core/groupby.py

```diff
@@ -2454,7 +2562,10 @@ def transform(self, func, *args, **kwargs):
 
         # if string function
         if isinstance(func, compat.string_types):
-            return self._transform_fast(lambda : getattr(self, func)(*args, **kwargs))
+            if func in _cython_transforms:
```

Comment:
`_intercept_cython` handles convert things like `np.sum->'sum'` but I do actually need to update it.

I'll try and think of way to make this more transparent too.